### PR TITLE
🐛 PRTL-1489 project, not case, filter for case entity ssm table

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

**Comment by alex-wilmer** [comment]
This is kind of hackish, I'm passing in a custom total value to the ssms table only in the case entity page. Very open to suggestions for a different strategy.

**Comment by Jephuff** [comment]
image

**Comment by Jephuff** [comment]
I'm a little confused. Isn't the problem that that table should be filtered by case_id and it's currently filtered by project? would passing in defaultFault filters as case_id and then adding a contextFilters for getting the filteredCases work?

**Comment by alex-wilmer** [comment]
>< right I needed to do the inverse of what I did kind of

**Comment by alex-wilmer** [comment]
oh I think it might need more change than that since `score` is being used for that column. maybe instead of passing additional filters, pass in a context component

**Comment by alex-wilmer** [comment]
or maybe even export a slightly modified version of the table model

**Comment by alex-wilmer** [comment]
@Jephuff okay I think I implemented what you suggested and I think this is good now.

**Comment by phuongmy** [comment]
Tested the PR just now and it looks good, great job, thanks Alex!

File: src/packages/@ncigdc/routes/CaseRoute.js [hunk 1]
```diff
@@ -18,16 +18,23 @@ import {
...
-  />,
+const SsmsTable = createCaseSummary(
+  ({
+    viewer,
+    node = viewer.repository.cases.hits.edges[0].node,
+    projectId = node.project.project_id,
+    caseId = node.case_id,
+  }) =>
+    <ST
+      defaultFilters={makeFilter([{ field: 'cases.case_id', value: caseId }])}
```
Comment (Jephuff) [inline src/packages/@ncigdc/routes/CaseRoute.js]: I think this filter is passed in already on line 110.

Comment (alex-wilmer) [inline src/packages/@ncigdc/routes/CaseRoute.js]: ah right since I switched the filters around. i'll remove it

**Pull request merged**